src/siteConstants.js

```diff
@@ -1,7 +1,5 @@
 /* This file includes the constants necessary to power the API homepage */
 export const PORT = process.env.PORT || 8080;
 export const API_ROUTE = process.env.HEROKU ? 'https://igboapi.com' : `http://localhost:${PORT}`;
-export const DICTIONARY_APP_URL = process.env.HEROKU
-  ? 'https://nkowaokwu.com'
-  : 'http://localhost:8000'; // A local instance of the dictionary app must be running
+export const DICTIONARY_APP_URL = 'https://nkowaokwu.com';
```

Comment:
We probably want to switch this back to what it was before since it handles the server side part. But inside `Demo.js`, you'll want to move your `setRedirectUrl` into the `if (typeof window !== 'undefined')` block so that you can change the link again on the client side